mlir/lib/Conversion/MIGraphXToTosa/MIGraphXToTosa.cpp

```diff
@@ -620,8 +622,8 @@ class DivConverter final : public OpConversionPattern<migraphx::DivOp> {
   matchAndRewrite(migraphx::DivOp op, OpAdaptor adaptor,
                   ConversionPatternRewriter &rewriter) const final {
     Location loc = op.getLoc();
-    TypedValue<TensorType> inATensor = op.getInA();
-    TypedValue<TensorType> inBTensor = op.getInB();
+    auto inATensor = cast<TypedValue<TensorType>>(op.getInA());
```

Comment:
Did they revert the automatic `typedValue` thing? Or is this just something with how tensors are defined upstream now?